### PR TITLE
Feat: Added post-features

### PR DESCRIPTION
**Pull request by RahulSingh9131**
This branch contains
* single Post page
* commenting on a post feature
* upvote & downvote feature
* and some refactoring

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
- perfect use of type script 
-  i really like your well  structured and consistent pattern in your code 
- good to merge

File: atoms/postsAtom.ts [hunk 1]
```diff
@@ -15,14 +15,23 @@ export type Post={
     createdAt:Timestamp;
 }
 
+export type PostVote={
+    id:string;
+    postId:string;
+    clubId:string;
+    voteValue:number;
+}
```
Comment (rahulcs754) [inline atoms/postsAtom.ts:23]: nice to use type script

**Review by rahulcs754 — APPROVED** [review]
the code looks and clearn

**Review by rohit8686 — APPROVED** [review]
- Clean code
- Nice use of TS
- Good to merge

**Pull request merged**